src/open_inwoner/components/templates/components/FilterBar/MultiSelectListbox.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n l10n form_tags icon_tags %}
+
+<div class="filter-bar__multiselect-listbox multiselect-listbox">
+    <button id="selectButton" type="button" class="button button__select" aria-haspopup="listbox" aria-expanded="false">
+        {% trans 'Status' %}:
+        {% icon icon="expand_more" icon_position="after" icon_outlined=True %}
+    </button>
+    <div id="listboxDropdown" class="multiselect-listbox__content" role="listbox" aria-labelledby="selectButton">
+        <div class="multiselect-listbox__scroll" role="presentation">
+            {% for option in statusfrequencies %}
```

Comment:
`statusfrequencies` is a list of tuples representing status and frequency. Instead of indexing into the tuples (`option.0`, `option.1`), which is not very readable, you can unpack the tuples into two variables:
```
{% for status, frequency in statusfrequencies %}
```
You can then use `status` and `frequency` instead of `option.0` and `option.1`

Reply:
Neat. Although the naming makes the component very specific in its use for Statuses only, not sure how to give it longevity yet. But for now this works.